src/main/java/hudson/scm/CompareAgainstBaselineCallable.java

```diff
@@ -54,7 +57,9 @@ public ClassLoader getClassLoader() {
      * so
      */
     public PollingResult call() throws IOException {
-        listener.getLogger().println("Received SCM poll call on " + nodeName + " for " + projectName + " on " + DateFormat.getDateTimeInstance().format(new Date()) );
+        String nodeLink = ModelHyperlinkNote.encodeTo("".equals(nodeName) ? ((Node) Jenkins.getInstance()) : Jenkins.getInstance().getNode(nodeName));
```

Comment:
This can be run on the agent JVM, in which case this code will throw NPEs or worse. You would need to do the note encoding on the master side and pass the result as `nodeName`. Ditto for `projectName`.